src/renault_api/renault_vehicle.py

```diff
@@ -336,6 +336,14 @@ async def get_hvac_sessions(
             response.get_attributes(schemas.KamereonVehicleHvacSessionsDataSchema),
         )
 
+    async def get_contracts(self):
```

Comment:
Needs a return type. Preferably in `renault_session` we return Response dataclass, and in `renault_vehicle` we return non-response dataclass.
```suggestion
    async def get_contracts(self) -> List[KameronContract]:
```